app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -325,7 +411,7 @@ public void run() {
 
     // WidgetManagerDelegate
     @Override
-    public void addWidget(final WidgetPlacement aPlacement, final WidgetAddCallback aCallback) {
+    public void addWidget(final WidgetPlacement aPlacement, final boolean aVisible, final WidgetAddCallback aCallback) {
```

Comment:
Why isn't aVisible part of WidgetPlacement? Okay, I see it is for widgets that don't have a placement struct. We should just create all widgets from the Java side. This is one of the things I wanted to talk about next week. Having it separate for now is probably fine. We can fix with the refactor.